modules/tinymce/changelog.txt

```diff
@@ -1,31 +1,31 @@
 Version 5.5.0 (TBD)
     Added a TypeScript declaration file to the bundle output for TinyMCE core #TINY-3785
-    Added new `table_column_resizing` setting to control how the table is affected when resizing columns using the resize bars #TINY-6001
-    Added ability to load TinyMCE in inline mode within a ShadowRoot #TINY-6128
+    Added new `table_column_resizing` setting to control how columns are resized when resizing tables using the resize bars #TINY-6001
+    Added the ability to load TinyMCE in inline mode within a ShadowRoot #TINY-6128
     Added the ability to remove images on a failed upload using the `images_upload_handler` failure callback #TINY-6011
     Added `hasPlugin` function to the editor API to determine if a plugin exists or not #TINY-766
-    Added new `ToggleToolbarDrawer` command and query state handler to allow the toolbar drawer to be programmatically toggled and its toggle state to be checked #TINY-6032
+    Added new `ToggleToolbarDrawer` command and query state handler to allow the toolbar drawer to be programmatically toggled and the toggle state to be checked #TINY-6032
     Added the ability to delete and navigate HTML media elements without the media plugin #TINY-4211
     Added `fullscreen_native` setting to the fullscreen plugin to enable use of the entire monitor #TINY-6284
-    Added table related oxide variables to the Style API for more granular control over table cell selection appearance #TINY-6311
-    Added new `toolbar_persist` setting to control the visibility of the inline toolbar. #TINY-4847
-    Added new APIs to allow for programmatic control of the inline toolbar visibility. #TINY-4847
+    Added table related oxide variables to the internal style sheets for more granular control over table cell selection appearance #TINY-6311
+    Added new `toolbar_persist` setting to control the visibility of the inline toolbar #TINY-4847
+    Added new APIs to allow for programmatic control of the inline toolbar visibility #TINY-4847
     Added the `origin` property to the `ObjectResized` and `ObjectResizeStart` events, to specify which handle the resize was preformed on #TINY-6242
     Added new StyleSheetLoader `unload` and `unloadAll` APIs to allow loaded stylesheets to be removed #TINY-3926
     Added the `LineHeight` query command and action to the editor #TINY-4843
     Added the `lineheight` toolbar and menu items, and added `lineheight` to the default format menu #TINY-4843
     Added a new `contextmenu_avoid_overlap` setting to allow contextmenus to avoid overlapping matched nodes #TINY-6036
-    Added new `listbox` dialog UI component for rendering a `selectbox` like list that accepts nested items #TINY-2236
+    Added new `listbox` dialog UI component for rendering a `selectbox` list that accepts nested items #TINY-2236
```

Comment:
This change is incorrect, as it's not rendering a `selectbox` list, it's rendering something that is similar.

Reply:
```suggestion
    Added new `listbox` dialog UI component for rendering a `selectbox`-styled list that accepts nested items #TINY-2236
```

Reply:
It's not styled like a selectbox list, it behaves similar to a selectbox list (eg has a dropdown that lets you select a certain value).

Reply:
It's styled like regular menus to clarify that point, whereas a selectbox's rendering is controlled via the browser.

Reply:
Should we just not mention `selectbox`? It's probably going to confuse people unless we put far more explanation...

Reply:
We need to describe what it is and a `selectbox`-like component is the easiest.

Reply:
```suggestion
    Added new `listbox` dialog UI component for rendering a selection menu that accepts nested items #TINY-2236
```

Reply:
@lnewson @metricjs :arrow_up:

Reply:
`selection menu` feels wrong, I can't quite explain why though...

Reply:
Just "menu" then?

Reply:
I still prefer the original string than this suggestion sorry.

Reply:
A menu doesn't make sense for a dialog form component. Can we steal anything from the docs we did for this? https://github.com/tinymce/tinymce-docs/blob/feature/TinyMCE_5.5/ui-components/dialogcomponents.md#listbox

Reply:
Maybe something like this?
> Added new `listbox` dialog UI component for rendering a menu-like dropdown that allows nested options #TINY-2236

Reply:
```suggestion
    Added new listbox dialog UI component for rendering a dropdown that allows nested options #TINY-2236
```